context/app/api/client.py

```diff
@@ -56,7 +56,17 @@ def get_entity(self, uuid):
                 'description': 'Mock Entity'
             }
 
-        query = {'query': {'ids': {'values': [uuid]}}}
+        if uuid is not None and hbm_id is not None:
+            raise Exception('Either UUID or HBM ID should be provided')
```

Comment:
We want to raise an exception when both values exist? Or do we want to raise an exception when neither values exist?

Reply:
If neither is provided, you'll get an error downstream... If both are provided, the function is being misused, and that's what this error is for. Rephrase error message? `Only UUID or HBM ID should be provided, not both`?